### PR TITLE
feat: adding bitbucket server docs

### PR DESCRIPTION
**Pull request by darmalovan**
Please Do not Merge until we release BB server to everyone.

### CODE REVIEW

File: _docs_aws-lambda/1_connecting-to-aws-lambda.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Connecting Snyk to AWS Lambda
 ---
-In order for Snyk to be able to monitor your deployed AWS Lambda functions, you'll first need to connect Snyk to your AWS Lambda account. You can do this by navigating to the [Integrations page](https://snyk.io/integrations) and clicking on "Connect to Connect to AWS Lambda".
+In order for Snyk to be able to monitor your deployed AWS Lambda functions, you'll first need to connect Snyk to your AWS Lambda account. You can do this by navigating to the [Integrations page](https://snyk.io/integrations) and clicking on "Connect to AWS Lambda".
```
Comment (darmalovan) [inline _docs_aws-lambda/1_connecting-to-aws-lambda.md]: Just a duplication typo

File: _docs_aws-lambda/3_adding-lambda-functions.md [hunk 2]
```diff
@@ -5,10 +5,10 @@ Once you've successfully connected Snyk to your AWS Lambda account, you'll be ab
 
 In either case, you'll see a list of any available projects on the AWS Lambda account you connected. Select the ones you want to monitor and click the "add to Snyk" button. 
 
-![Screenshot of the screen displaying the available AWS Lambda apps to monitor](https://res.cloudinary.com/snyk/image/upload/w_auto,c_scale,q_auto/v1493173602/serverless-docs/aws-projects.png)
+![Screenshot of the screen displaying the available AWS Lambda apps to monitor](https://res.cloudinary.com/snyk/image/upload/w_auto,c_scale,q_auto/v1493173601/serverless-docs/aws-functions-to-test.png)
```
Comment (darmalovan) [inline _docs_aws-lambda/3_adding-lambda-functions.md]: The order of the pictures was the other way around

File: _docs_bitbucket-server/00_intro.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: ""
+---
+
+Snyk's Bitbucket Server integration lets you monitor the source code of your Node.js and Rubygems repos for any known vulnerabilities found in the application's dependencies, testing at a frequency you control.
+
+This integration will works only for publicly available Bitbucket Server instances and not for bitbucket.org.
```
Comment (joshje) [inline _docs_bitbucket-server/00_intro.md]: Should be:
> This integration only works with Bitbucket Server instances that are publicly reachable (not on a private network) and not for bitbucket.org.

File: _docs_bitbucket-server/00_intro.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+For each test, Snyk will communicate directly with Bitbucket Server to determine exactly what code is currently pushed and what dependencies are being used. Each dependency will in turn be tested against Snyk's vulnerability database to see if it contains any known vulnerabilities. 
+
+If vulnerabilities are found, you will be alerted (via email or Slack) so that you can take immediate action.
+
+In order to turn on the Bitbucket Server integration you'll need to:
+
+1. [Connect to Bitbucket Server](#connecting-snyk-to-bitbucket-server) from the integrations page
+2. [Add your Bitbucket Server account credentials](#generating-your-bitbucket-user) to Snyk
```
Comment (joshje) [inline _docs_bitbucket-server/00_intro.md]: This link is broken

File: _docs_bitbucket-server/00_intro.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+3. [Select the projects you want to monitor](#adding-bitbucket-server-repos) and click "Add to Snyk"
```
Comment (joshje) [inline _docs_bitbucket-server/00_intro.md]: This link is also broken

File: _docs_bitbucket-server/2_generating-your-bitbucket-user.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+title: Generating your Bitbucket Server Credentials
+---
+To give Snyk access to your Bitbucket Server account, we recommend setting up a dedicated user with read-only permissions.
+
+You can create a user from the admin panel on your bitbucket server.
```
Comment (tkadlec) [inline _docs_bitbucket-server/2_generating-your-bitbucket-user.md]: bitbucket server -> Bitbucket server

**Pull request merged**